PIL/Image.py

```diff
@@ -632,6 +677,7 @@ def _repr_png_(self):
 
     @property
     def __array_interface__(self):
+        # type: () -> Dict[Text, Any]
```

Comment:
Do you mean that to be a Union? From the docs only, and given that this needs to potentially interoperate with other code which exposes all the dictionary keys, I'd imagine this at least needs to include `None` and `List` as well?

This may be where it's sometimes easier to use `Any`, at least initially ;)